chaos_genius/alerts/email.py

```diff
@@ -74,81 +80,51 @@ def send_email(recipient_emails, message, count=0):
         else:
             toaddr = recipient_emails
 
-        server = init_smtp_server()
-        server.sendmail(EMAIL_SENDER, toaddr, message.as_string())
+        server = init_smtp_server(host, port, user, password)
+        server.sendmail(sender, toaddr, message.as_string())
         server.quit()
-        print("Email sent to " + ", ".join(toaddr))
+        logger.info("Email sent to " + ", ".join(toaddr))
     except smtplib.SMTPServerDisconnected:
-        print(f"Retry ({count}) for the email")
+        logger.info(f"Retry ({count}) for the email")
         if count < 3:
-            send_email(recipient_emails, message, count)
+            send_email(
+                recipient_emails, message, host, port, user, password, sender, count
+            )
         else:
-            print("Email Sending Failed after max retries")
-    except Exception:
-        print(traceback.format_exc())
-
-
-def initialize_env_variables():
-    global EMAIL_HOST
-    global EMAIL_HOST_PORT
-    global EMAIL_HOST_USER
-    global EMAIL_HOST_PASSWORD
-    global EMAIL_SENDER
-    global DEBUG
-    creds = get_creds("email")
-    (
-        EMAIL_HOST,
-        EMAIL_HOST_PORT,
-        EMAIL_HOST_USER,
-        EMAIL_HOST_PASSWORD,
-        EMAIL_SENDER,
-    ) = creds
+            logger.error("Email Sending Failed after max retries")
+            raise
 
 
 def send_static_alert_email(
-    recipient_emails, subject, messsage_body, alert_info, files=[]
-):
-    """Send the static event alert email with the CSV attachment
+    recipient_emails: List[str], subject: str, messsage_body: str, files=[]
```

Comment:
yeah, it's fine here since it's not mutated. i'll annotate it's type as `Sequence[Dict]` to ensure that mutation will cause a type error in the editor.